### ISSUE TITLE
`?imgRaster` should offer a choice between the SpatialExperiment and SpatialImage methods

### ISSUE THREAD

**Issue by LTLA** [drighelli/SpatialExperiment#111]
I don't know how many other generics have methods for both classes, but methods for both classes should respond to `?<method-name>`. Otherwise users have to type out `?"imgRaster,SpatialExperiment-method"`, which is not obvious.

**Comment by LTLA** [comment]
On a related note, `?imgSource` doesn't appear to be actually documented.

**Comment by HelenaLC** [comment]
Yeah, I found this not straightforward and really confusing to document actually... Could you clarify what you mean by "should respond to `?<method-name>`"? I.e., what behavior would you expect / want to be supported? I think the issue is that most people (including me) will just do `?imgX`, and it's not clear whether this should point to the SPI or SPE's `imgData-methods` doc. And yes, good point re `imgSource`; it's currently only documented under `?`imgData-methods`...

**Comment by HelenaLC** [comment]
I think this doesn't work. The above prompt upon `?<method>` appears when multiple packages export `<method>`. However, when I add `@alises imgRaster` to the `imgData-methods` help page, `?imgRaster` still goes directly to the `SpatialImage-class` one. Looks like having the same alias appear in multiple docs is ignored, and the first doc (whatever first means here) is shown silently.

**Comment by LTLA** [comment]
Hm. In that case, it may be worth having an explicit page for the `img*` generic, so that anyone typing `?imgRaster` will get to the documentation for the method rather than the class.

In general, it seems like we should document by generic, rather than by class, with the assumption that someone doing `?generic` would prefer to know the generic rather than one of the classes it operates on.

The exception would be for generics that only apply to a single class, which can be grouped in the docpage for that class.
